catalog/sync_test.go

```diff
@@ -19,44 +19,47 @@ import (
 	"github.com/hashicorp/consul-aws/internal/flags"
 )
 
+// TestSync is an integration test that creates a service in Consul and AWS.
+// Documentation on setup can be found in the engineering docs for `consul-aws`.
 func TestSync(t *testing.T) {
 	if len(os.Getenv("INTTEST")) == 0 {
 		t.Skip("Set INTTEST=1 to enable integration tests")
 	}
 	namespaceID := os.Getenv("NAMESPACEID")
-	if len(namespaceID) == 0 {
-		namespaceID = "ns-n5qqli2346hqood4"
+	if namespaceID == "" {
+		t.Fatalf("The NAMESPACEID varaible must be set.")
 	}
 	runSyncTest(t, namespaceID)
 }
 
 func runSyncTest(t *testing.T, namespaceID string) {
+	// Test Setup
 	config, err := awsconfig.LoadDefaultConfig(context.TODO())
 	if err != nil {
 		t.Fatalf("Error retrieving AWS session: %s", err)
 	}
-	a := awssd.NewFromConfig(config)
+	awssdClient := awssd.NewFromConfig(config)
 
 	f := flags.HTTPFlags{}
-	c, err := f.APIClient()
+	consulClient, err := f.APIClient()
 	if err != nil {
 		t.Fatalf("Error connecting to Consul agent: %s", err)
 	}
 
-	cID := "r1"
-	cName := "redis"
-	aName := "web"
+	consulServiceID := "r1"
+	consulServiceName := "redis"
+	awsServiceName := "web"
```

Comment:
I just hated these variable names because it's hard to debug without flipping back and forth.